### PR TITLE
[draft] Try Vue3 compat mode

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Just spent an hour hacking on this with @fschade, we got it to build successfully but didn't manage to get the Vue instance to accept the (new) translations plugin. Saving this as WIP so anyone can try and continue

## Related Issue
- Targets #5268

## Motivation and Context
See issue

## Types of changes
- [X] Technical debt

## Open tasks:
- [ ] Update the way Vue + vuex are initialized
- [ ] Fix all deprecation warnings
- [ ] Update dependencies, remove dependencies

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -162,11 +174,6 @@ if (process.env.SERVER === 'true') {
       port: process.env.PORT || 9100
     })
   )
-  plugins.push(
-    livereload({
-      watch: 'dist'
-    })
-  )
```
Comment (dschmidt) [inline rollup.config.js:169]: Why did you remove livereload?

If you remove it here for a reason, then the imports and references in package.json and yarn.lock should be removed as well.

Comment (pascalwengerter) [reply rollup.config.js:169]: Hey, thanks for the hint and yes technically you're right. This is/was a draft PR and something about livereload broke the build process (and we didn't bother investigating there since it was a hacky "let's see how far we get" anyways)

Will get fixed in the cleanup if/once we continue with this PR ;)

**Comment by pascalwengerter** [comment]
Please nobody put any effort into this before #5752 has been merged - I'll rebase it then and we can continue working on it with a way better setup

**Comment by pascalwengerter** [comment]
> Please nobody put any effort into this before #5752 has been merged - I'll rebase it then and we can continue working on it with a way better setup

Rebased this branch, doesn't build anymore with what I expect to be a postcss error. @fschade can you maybe take a look?

**Comment by pascalwengerter** [comment]
I'll close this as obsolete, it is linked in the relevant Vue3-update-issues but most likely will be superseded by a new PR - looking forward!
